util/src/types.rs

```diff
@@ -1,104 +1 @@
-// Copyright 2019 The Grin Developers
-//
-// Licensed under the Apache License, Version 2.0 (the "License");
-// you may not use this file except in compliance with the License.
-// You may obtain a copy of the License at
-//
-//     http://www.apache.org/licenses/LICENSE-2.0
-//
-// Unless required by applicable law or agreed to in writing, software
-// distributed under the License is distributed on an "AS IS" BASIS,
-// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
-// See the License for the specific language governing permissions and
-// limitations under the License.
 
-//! Logging configuration types
-
-/// Log level types
-#[derive(Debug, Clone, Serialize, Deserialize, PartialEq)]
-pub enum LogLevel {
-	/// Error
-	Error,
-	/// Warning
-	Warning,
-	/// Info
-	Info,
-	/// Debug
-	Debug,
-	/// Trace
-	Trace,
-}
-
-/// 32 log files to rotate over by default
-pub const DEFAULT_ROTATE_LOG_FILES: u32 = 32 as u32;
-
-/// Logging config
-#[derive(Debug, Clone, Serialize, Deserialize, PartialEq)]
-pub struct LoggingConfig {
-	/// whether to log to stdout
-	pub log_to_stdout: bool,
-	/// logging level for stdout
-	pub stdout_log_level: LogLevel,
-	/// whether to log to file
-	pub log_to_file: bool,
-	/// log file level
-	pub file_log_level: LogLevel,
-	/// Log file path
-	pub log_file_path: String,
-	/// Whether to append to log or replace
-	pub log_file_append: bool,
-	/// Size of the log in bytes to rotate over (optional)
-	pub log_max_size: Option<u64>,
-	/// Number of the log files to rotate over (optional)
-	pub log_max_files: Option<u32>,
-	/// Whether the tui is running (optional)
-	pub tui_running: Option<bool>,
-}
-
-impl Default for LoggingConfig {
-	fn default() -> LoggingConfig {
-		LoggingConfig {
-			log_to_stdout: true,
-			stdout_log_level: LogLevel::Warning,
-			log_to_file: true,
-			file_log_level: LogLevel::Info,
-			log_file_path: String::from("grin.log"),
-			log_file_append: true,
-			log_max_size: Some(1024 * 1024 * 16), // 16 megabytes default
-			log_max_files: Some(DEFAULT_ROTATE_LOG_FILES),
-			tui_running: None,
-		}
-	}
-}
-
-use std::ops::Deref;
-use zeroize::Zeroize;
-/// Zeroing string, mainly useful for password
-#[derive(Clone, PartialEq, PartialOrd)]
-pub struct ZeroingString(String);
```

Comment:
This seems unrelated? And its password related (masking out passwords in the logs?) So probably best to have this in a separate PR if this is indeed unused and to be cleaned up.

Reply:
Yes it's unrelated and unused. Will put it back for now.